Update User Guide - Note for COG layer (#9662)
geosolutions-it/MapStore2: docs/user-guide/catalog.md

!!! Note
    To properly display COG layers in your MapStore map, it is necessary to add the reference system definition supported by the COG in the MapStore [projectionDefs configuration](../../developer-guide/local-config/#projectiondefs-configuration)

!!! warning

>     The feature is currently in experimental state, and won't be available in the default service types list of the catalog.

Reviewer: ```suggestion
     COG catalog types in MapStore is still in experimental state and won't be available in the default service types list of the Catalog tool.
```

>     In order to enable this service, update the [MetadataExplorer plugin's configuration](https://mapstore.geosolutionsgroup.com/mapstore/docs/api/plugins#plugins.MetadataExplorer) in `localConfig.json` as shown below

Reviewer: ```suggestion
    In order to enable this service, update the default [Catalog tool configuration](https://mapstore.geosolutionsgroup.com/mapstore/docs/api/plugins#plugins.MetadataExplorer) in `localConfig.json`  or inside the application context wizard as shown below:
```

    ```diff
    {
        "name": "MetadataExplorer",
        "cfg": {
            ...
            serviceTypes: [
                ...
    +           { name: "cog", label: "COG" }
            ]
        }
    }
    ```

#### Advanced Settings

In addition to the standard options, only for COG catalog sources, through the **Advanced Settings** the user can configure also the following option:

<img src="../img/catalog/advanced_settings_cog.jpg" class="ms-docimage"  style="max-width:600px;"/>

* *Download file metadata on search*: this option will fetch metadata to support the zoom to layer when the layer is added to the [TOC](toc.md#table-of-contents).

> !!! warning

Reviewer: ```suggestion
!!! Note
```

>     For now, the implementation of the COG catalog on MapStore is experimental and the features currently available for the COG layers are:

Reviewer: ```suggestion
    The tool capabilities currently available for the COG layers are:
```
